matchms/networking/SimilarityNetwork.py

```diff
@@ -0,0 +1,135 @@
+import networkx as nx
+import numpy
+from matchms import Scores
+from .networking_functions import get_top_hits
+
+
+class SimilarityNetwork:
+    """Create a spectal network from spectrum similarities.
+
+    For example
+
+    .. testcode::
+
+        import numpy as np
+        from matchms import Spectrum, calculate_scores
+        from matchms.similarity import ModifiedCosine
+        from matchms.networking import SimilarityNetwork
+
+        spectrum_1 = Spectrum(mz=np.array([100, 150, 200.]),
+                              intensities=np.array([0.7, 0.2, 0.1]),
+                              metadata={"precursor_mz": 100.0,
+                                        "testID": "one"})
+        spectrum_2 = Spectrum(mz=np.array([104.9, 140, 190.]),
+                              intensities=np.array([0.4, 0.2, 0.1]),
+                              metadata={"precursor_mz": 105.0,
+                                        "testID": "two"})
+
+        # Use factory to construct a similarity function
+        modified_cosine = ModifiedCosine(tolerance=0.2)
+        spectrums = [spectrum_1, spectrum_2]
+        scores = calculate_scores(spectrums, spectrums, modified_cosine)
+        ms_network = SimilarityNetwork(identifier="testID")
+        ms_network.create_network(scores)
+
+        print("Nodes:", ms_network.graph.nodes())
+
+    Should output
+
+    .. testoutput::
+
+        Nodes: ['one', 'two']
```

Comment:
When I run locally with `cd readthedocs && make doctest` the test failed with
```
Expected:
    Nodes: ['one', 'two']
Got:
    Nodes: ['two', 'one']
```

Use sort() so order is deterministic
```suggestion
        sort(ms_network.graph.nodes())

    Should output

    .. testoutput::

        ['one', 'two']
```